src/AppInstallerCLICore/VTSupport.h

```diff
@@ -133,6 +134,20 @@ namespace AppInstaller::CLI::VirtualTerminal
         extern const Sequence EraseLineBackward;
         extern const Sequence EraseLineEntirely;
     }
+
+    namespace Progress
+    {
+        enum class ProgressState

```

Comment:
```suggestion
        enum class State
```

To reduce repetition so that it is just `Progress::State`

Reply:
Applied your suggestion in c4d5c48a1c5a638d15025687a060f752bd5be3f0.